### DISCUSSION TITLE
Export kriging as raster

### DISCUSSION THREAD

**Discussion by patricia-oliveira-gaivota** [GeoStat-Framework/PyKrige#241] [Q&A]
Hello, everyone!
I am following this tutorial here to do kriging to Brazil temperature: https://geostat-framework.readthedocs.io/projects/gstools/en/stable/examples/08_geo_coordinates/01_dwd_krige.html#
I did Universal kriging and plot the map exactly as the code explains. But now I want to export the interpolation result as a raster. How can I do this?

**Comment by MuellerSeb** [comment]
Hi there,

Take a look at [write_asc_grid](https://geostat-framework.readthedocs.io/projects/pykrige/en/stable/generated/pykrige.kriging_tools.write_asc_grid.html#pykrige.kriging_tools.write_asc_grid) to export raster files.

Sebastian
